diff --git a/fix-v3.3/cdcacm.c b/fix-v3.3/cdcacm.c
index 9543b19..7a7e7ef 100644
--- a/fix-v3.3/cdcacm.c
+++ b/fix-v3.3/cdcacm.c
@@ -881,6 +881,17 @@ static int acm_write_buffers_alloc(struct acm *acm)
 	return 0;
 }
 
+
+// include/linux/pm.h#L433
+#define PMSG_IS_AUTO(msg)       (((msg).event & PM_EVENT_AUTO) != 0)
+
+// include/uapi/linux/usb/ch9.h#L603
+static inline int usb_endpoint_maxp(const struct usb_endpoint_descriptor *epd)
+{
+	return __le16_to_cpu(epd->wMaxPacketSize);
+}
+
+
 static int acm_probe(struct usb_interface *intf,
 		     const struct usb_device_id *id)
 {
